ci-operator/jobs/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-master-presubmits.yaml

```diff
@@ -74,6 +74,70 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws-operator,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
```

Comment:
Please add `optional: true` as well. `always_run: false` means the job will not get run by default. Once you type `/test e2e-aws-operator-tech-preview` in a comment, it will get run and will be still required.

Reply:
done